### PR TITLE
Slides: make list sortable with drag and drop

### PR DESCRIPTION
**Pull request by rwaldron**

### CODE REVIEW

**Comment by rwaldron** [comment]
> nice! next up re-order slide components with drag?

Hm. Maybe!

File: client/components/Editor/Slides.jsx [hunk 1]
```diff
@@ -148,12 +153,19 @@ class Slides extends React.Component {
         this.setState({ currentSlideIndex });
     }
 
+    async onChangeSlideOrder(...args) {
+        await this.moveSlide(
```
Comment (evmiguel) [inline client/components/Editor/Slides.jsx]: Nice!

File: client/components/Editor/Slides.jsx [hunk 2]
```diff
@@ -169,72 +181,50 @@ class Slides extends React.Component {
...
-                                className="Slides-slide-sidebar-container"
-                            >
-                                <Card
-                                    className="Slides-slide-sidebar-card"
-                                    onClick={() =>
-                                        this.setState({
-                                            currentSlideIndex: index
-                                        })
-                                    }
+                        <Sortable onChange={onChangeSlideOrder}>
```
Comment (evmiguel) [inline client/components/Editor/Slides.jsx]: Cool component!

Comment (rwaldron) [reply client/components/Editor/Slides.jsx]: It only took me 4 tries...

image

**Pull request merged**